solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -1047,6 +1097,18 @@ contract RandomBeaconGovernance is Ownable {
             );
     }
 
+    function getRemainingAuthorizationDecreaseDelayUpdateTime()
```

Comment:
Minor thing, could we add a description to this function if the other functions have it?

Reply:
Good point but please let me address it in a followup PR - I will have one tomorrow and don't want to rebase everything again 😬